starters/gatsby-starter/src/components/seo.jsx

```diff
@@ -1,3 +1,5 @@
+import React from 'react';
```

Comment:
Can we remove this import?

Reply:
Importing React like this helps the compiler on .jsx and .tsx files even if we aren't directly using it. I'm not sure in what circumstances it's actually necessary. In this case, I added it in order to fix a gatsby build error.
@Aumkar

Reply:
Ok, makes sense